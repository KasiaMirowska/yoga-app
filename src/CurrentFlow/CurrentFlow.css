.current-flow {
    padding-top: 20px;
    padding-bottom: 20px;
}