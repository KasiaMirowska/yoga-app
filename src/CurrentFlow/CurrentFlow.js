import React from 'react';
import YogaContext from '../Context';
import './CurrentFlow.css';

export default function CurrentFlow() {
    
    return (
        <YogaContext.Consumer>
            {(value) => {
                return(
                    <div className='current-flow'>
                        <h2 className='title'>Current flow: {`${value.currentFlow.title}`.toUpperCase()}</h2>
                    </div>
                )
            }}    
               
                
        
    
               </YogaContext.Consumer>
    )
}