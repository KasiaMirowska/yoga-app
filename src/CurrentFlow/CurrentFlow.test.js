import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import CurrentFlow from './CurrentFlow';
import {BrowserRouter as BR} from 'react-router-dom';

describe('CurrentFlow component', () => {
    
    it ('renders without crashing',() => {
        const div = document.createElement('div');
        ReactDom.render(<BR><CurrentFlow /></BR>, div);
        ReactDom.unmountComponentAtNode(div);
    });
    it('renders UI as expected', () => {
        const loginPage = renderer.create(<BR><CurrentFlow /></BR>);
        expect(loginPage.toJSON()).toMatchSnapshot();
    })

})