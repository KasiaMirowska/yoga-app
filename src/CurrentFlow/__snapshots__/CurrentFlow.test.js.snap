// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentFlow component renders UI as expected 1`] = `
<div
  className="current-flow"
>
  <h2
    className="title"
  >
    Current flow: 
    UNDEFINED
  </h2>
</div>
`;
